.github/workflows/indexer-sync.yml

```diff
@@ -0,0 +1,217 @@
+---
+name: Indexer Sync Automation
+
+on:
+  schedule:
+    # Run daily at 2 AM UTC
+    - cron: '0 2 * * *'
+  workflow_dispatch:
+    inputs:
+      force_sync:
+        description: 'Force sync even if up to date'
+        required: false
+        default: false
+        type: boolean
+
+jobs:
+  sync-indexers:
+    name: Sync Indexers from Jackett
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
```

Comment:
Any specific reason why it is running on such an old node version? given it is EoL